concepts/strings/about.md

```diff
@@ -17,6 +17,19 @@ escaped == verbatim;
 // => true
 ```
 
+At some point you might need to separate a string into multiple parts. This might be because you want to concatenate different parts from multiple strings or you 
+need to analyse a specific part more closely but don't need the rest. That happens mostly with application logs.
+
+The easiest way to do this is with the [Substring method][substring] 
```

Comment:
This text is slightly verbose. Maybe we could shorten this a bit? I also don't know if we should be listing reasons for using `Substring`.

What about:

```suggestion
If you only need a part of a string, you can use the [`Substring()` method][substring] to extract just that part:
```